### PR TITLE
Add views

### PR DESCRIPTION
**Pull request by butlermuwo**
## Blog App Views 
In this pull request I did the following:
- Implemented the design from the sneak peek wireframes.
- Used methods that I created in processing data in models task(pull request).
- In order to display more posts & comments, I added some data to my database.

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,13 @@
-<h1>home of users posts</h1>
\ No newline at end of file
+<h1><%= @user.name %></h1>
+<img src="<%= @user.photo %>" alt="<%= @user.name %>">
+<p><%= @user.bio %></p>
+<p>Number of Posts <%= @user.posts.count%></p>
+<h2>Posts</h2>
+<% @user.recent_posts.each do |post| %>
+    <div class="post">
+        <h3><%= link_to post.title, user_posts_path(post.author_id) %></h3>
```
Comment (Lameck1) [inline app/views/users/show.html.erb]: - [ ] Clicking on a single post should take us to that single post in the posts/show. Right now this is not the case, clicking on any post takes us to view all posts. Please consider refactoring like this:

  ```suggestion
        <h3><%= link_to post.title, user_post_path(post) %></h3>
  ```

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,13 @@
...
+        <p><%= post.text %></p>
+        <p>Comments: <%= post.comments.count %></p>
+        <p>Likes: <%= post.likes.count %></p>
+    </div>
+<% end %>
```
Comment (Lameck1) [inline app/views/users/show.html.erb]: - [ ] After displaying the three most recent posts, a user should be able to see a button with the name `See all posts`. Now, this where we should add a link to the path that shows all users posts.

  Please add this button and make it link to all user's posts. This is as suggested in the design section below:
  
  
  image

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1 +1,8 @@
-<h1> this is a list of posts of a given user </h1>
\ No newline at end of file
+<% @posts.each do |post|%>
+    <div class="post">
+        <h3><%= link_to post.title, {:action=> 'show', :id => post.id} %></h3>
+        <p><%= post.text %></p>
+        <p>Comments: <%= post.comments.count %></p>
+        <p>Likes: <%= post.likes.count %></p>
+    </div>
+<%end%>
```
Comment (Lameck1) [inline app/views/posts/index.html.erb]: - [ ] When we render a post, we should also render the five most recent comments of that post. That is suggested by this section of the wireframe 👇🏼:

  image
  
  Please add code that will display the five most recent comments in this view.

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi, @butlermuwo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Highlights
- Professional README ✔️
- Clean code ✔️ 
- Good commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**Please, remember to tag me in your question so I can receive the notification.**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/users/show.html.erb (hunk 1 above)
Comment (butlermuwo) [reply app/views/users/show.html.erb]: Thanks @Lameck1 done.

File: app/views/users/show.html.erb (hunk 2 above)
Comment (butlermuwo) [reply app/views/users/show.html.erb]: Thanks @Lameck1  Done.

File: app/views/posts/index.html.erb (hunk 3 above)
Comment (butlermuwo) [reply app/views/posts/index.html.erb]: Thanks @Lameck1 done.

**Review by rustam-tolipov — APPROVED** [review]
## Hi @butlermuwo 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by butlermuwo** [comment]
Cheers!😊 @Rustamxon7

**Pull request merged**